plugins/agent_types/AcquiaLiftTarget.inc

```diff
@@ -61,4 +61,20 @@ class AcquiaLiftTarget extends PersonalizeAgentBase implements PersonalizeExplic
   public function supportsMultipleDecisionPoints() {
     return FALSE;
   }
+
+  /**
+   * Implements PersonalizeAgentGoalInterface::useClientSideGoalDelivery().
+   *
+   * @return boolean
+   */
+  public function useClientSideGoalDelivery() {
+    return TRUE;
+  }
+
+  /**
+   * Implements PersonalizeAgentGoalInterface::sendGoal().
+   */
+  public function sendGoal($goal_name, $value = NULL) {
+    return;
+  }
```

Comment:
Heh, this is one of the changes I made in my branch and then realized you would already have made it. It also made me realize that these functions are totally unused, but I'll refrain from ripping out that interface too ;)